project.xml

```diff
@@ -41,6 +41,9 @@
         <option name = "use_checkout_explicit" value = "1" />
     </target>
 
+    <classfilename use-cxx = "true" source-extension = "cc" header-extension = "h"                                
```

Comment:
Enable gcc-4.9 may be needed for Travis (because of shm) =>  use-cxx-gcc-4-9 = "true"